lib/components/pinnable/scroll_view.dart

```diff
@@ -29,6 +29,9 @@ class PinnableMessageScrollView extends ScrollView {
 
   @override
   List<Widget> buildSlivers(BuildContext context) {
+    final brightness = MediaQuery.of(context).platformBrightness;
```

Comment:
Oh, does `Theme.of(context).brightness` work?

Reply:
Yes :D